docusaurus/docs/encryption.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: encryption
-title: @idearium/encryption
+title: '@idearium/encryption'
```

Comment:
Does this need `sidebar_label` and `slug`?

Reply:
@smebberson Doesn't look like it, it only needs to be there on the first page to make the sidebar show up.